pkg/lockfile/parse-pdm-lock.go

```diff
@@ -0,0 +1,80 @@
+package lockfile
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/BurntSushi/toml"
+)
+
+type PdmLockPackage struct {
+	Name     string   `toml:"name"`
+	Version  string   `toml:"version"`
+	Groups   []string `toml:"groups"`
+	Revision string   `toml:"revision"`
+}
+
+type PdmLockFile struct {
+	Version  string           `toml:"lock-version"`
+	Packages []PdmLockPackage `toml:"package"`
+}
+
+const PdmEcosystem = PipEcosystem
+const PdmLockFileName = "pdm.lock"
+const pdmGroupDefault = "default"
+const pdmGroupDev = "dev"
```

Comment:
we don't typically use constants for these, and you're only referencing them each once so lets just inline them

Reply:
Removed constants